src/docs/asciidoc/api-guide/resources/learning-outcomes-and-assessment-questions.adoc

```diff
@@ -28,3 +28,29 @@ Read more about our AI policy https://www.boclips.com/ai-policy[here]
 
 operation::assessment-questions-get[snippets='path-parameters,http-request,curl-request,http-response']
 
+=== Retrieving filtered highlights
+List of filtered highlights
+
+[IMPORTANT]
+====
+
+*Data provided by this endpoint is AI-generated, in BETA, not subject to our SLA policies, and format may change with short notice.*
```

Comment:
We strongly recommend adding the header `X-Boclips-Api-Version` with the current version `2024-04-24` to avoid breaking changes

Reply:
I dont know where the header info will show and if this will be duplicate information in the same place?